README.md

```diff
@@ -0,0 +1,197 @@
+
+Check out `pysdccc.download` or `pysdccc.download_async`. Otherwise, if `pysdccc[cli]` is installed, `$ pysdccc install https://url/to/sdccc.zip` can also be used.
+
+### Create configuration file
+
+Configure the test consumer. Check the [test consumer configuration](https://github.com/Draegerwerk/SDCcc/?tab=readme-ov-file#test-consumer-configuration) for more information.
+
+```python
+import pathlib
+import toml  # has to be installed by the user
+import pysdccc
+
+config = {
+    'SDCcc': {
+        ...  # add all relevant config parameter
+    }
+}
+config_path = pathlib.Path('/path/to/configuration/file.toml')
+config_path.write_text(toml.dumps(config))
+
+runner = pysdccc.SdcccRunner(
+    pathlib.Path('/path/to/sdccc/result/directory'),
+)
+
+runner.run(
+    config=config_path,
+    requirements=pathlib.Path('/path/to/requirements/file.toml'),
+)
+
+# or if you have already downloaded the version
+config = runner.get_config()  # load default configuration
+config['SDCcc']['Consumer']['DeviceEpr'] = "urn:uuid:12345678-1234-1234-1234-123456789012"  # e.g. change device epr
+# save and run as above
+```
+
+### Create requirements file
+
+Enable or disable specific requirements. Check the [test requirements](https://github.com/Draegerwerk/SDCcc/?tab=readme-ov-file#enabling-tests) for more information.
+
+```python
+import pathlib
+import toml  # has to be installed by the user
+import pysdccc
+
+requirements = {
+    'BICEPS': {
+        ...  # add all requirements to be tested
+    }
+}
+requirements_path = pathlib.Path('/path/to/configuration/file.toml')
+requirements_path.write_text(toml.dumps(requirements))
+
+runner = pysdccc.SdcccRunner(
+    pathlib.Path('/path/to/sdccc/result/directory'),
+)
+# optionally, check whether you did not add a requirement that is not available
+runner.check_requirements(requirements_path)
+runner.run(
+    config=pathlib.Path('/path/to/configuration/file.toml'),
+    requirements=requirements_path,
+)
+
+# or, if you have already downloaded the version
+requirements = runner.get_requirements()  # load default configuration
+requirements['BICEPS']['R0033'] = False  # e.g. disable biceps R0033
+# save and run as above
+```
+
+### Create test parameter configuration
+
+Some tests require individual parameters. Check the [test parameter configuration](https://github.com/Draegerwerk/SDCcc/?tab=readme-ov-file#test-parameter-configuration) for more information.
+
+```python
+import pathlib
+import toml  # has to be installed by the user
+import pysdccc
+
+config = {
+    'TestParameter': {
+        ...
+    }
+}
+requirements_path = pathlib.Path('/path/to/configuration/file.toml')
+requirements_path.write_text(toml.dumps(config))
+
+runner = pysdccc.SdcccRunner(
+    pathlib.Path('/path/to/sdccc/result/directory'),
+)
+runner.run(
+    config=pathlib.Path('/path/to/configuration/file.toml'),
+    requirements=pathlib.Path('/path/to/requirements/file.toml'),
+    testparam=requirements_path,
```

Comment:
this seems misleading to me, requirements and testsparam are pointing to a kind of requirements path.

Reply:
good catch. fixed